### DISCUSSION TITLE
Multiple <icon> tags?

### DISCUSSION THREAD

**Discussion by buhtz** [ximion/appstream#587] [Q&A] [answered]
Can I have multiple <icon> tags?
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-icon

The reason would be to define fallback icons. Starting with "locale" going to "remote" and "stock"

**Comment by ximion** [comment, marked as answer]
You shouldn't have to define any icon at all in a MetaInfo file. It should be pulled automatically from your desktop-entry file.

If you do have something that doesn't have a desktop-entry file, like a `console-application` defining `<icon type="stock">blah</icon>` is valid though, but make sure you actually install a stock icon then.
Defining a stock icon is also pretty much the only ubiquitously supported icon type for MetaInfo files.

You can also only have one icon per component in your MetaInfo file.
